itou/openid_connect/inclusion_connect/views.py

```diff
@@ -55,10 +55,14 @@ def bind_to_request(self, request):
         return request
 
 
-def _redirect_to_login_page_on_error(error_msg, request=None):
+def _redirect_to_login_page_on_error(error_msg=None, request=None, user_error_msg_sufix=""):
     if request:
-        messages.error(request, "Une erreur technique est survenue. Merci de recommencer.")
-    logger.error(error_msg)
+        message = "Une erreur technique est survenue. Merci de recommencer."
+        if user_error_msg_sufix:
+            message = " ".join([message, user_error_msg_sufix])
+        messages.error(request, message)
+    if error_msg:
+        logger.error(error_msg)
```

Comment:
La signature me parait bizarre maintenant. Le corps de la méthode semble avoir beaucoup de logique, pour simplement ne pas copier-coller “Une erreur technique est survenue. Merci de recommencer.”
Aussi, je pense que le log d’erreur devrait être émis dans tous les cas.

Aussi, préciser que les personnes peuvent contacter le support s’ils pensent avoir un problème avec l’outil me semble suffisamment évident pour ne pas le préciser, ce qui enlève le cas d’usage pour `user_error_msg_sufix`.

Reply:
Il faut qu'on réfléchisse à ce qu'il faut faire.
Cette PR a pour but de régler les nombreuses erreurs sentry :
https://sentry.io/organizations/itou/issues/3446848736/?project=6164438&query=is%3Aunresolved+assigned%3Ame&referrer=issue-stream&statsPeriod=14d
https://sentry.io/organizations/itou/issues/3582890365/?project=6164438&query=is%3Aunresolved+assigned%3Ame&referrer=issue-stream&statsPeriod=14d

- Si on considère qe c'est un PEBCAK (bookmark sur la page d'inclusion connect) ne souhaite-t-on pas les cacher dans sentry (d'où le logger qu'on retire) ?
- Veut-on inciter les utilisateurs à nous contacter pour voir si on n'a pas un autre soucis ?

Reply:
La `KeyError` (https://sentry.io/organizations/itou/issues/3582890365/?project=6164438&query=is%3Aunresolved+assigned%3Ame&referrer=issue-stream&statsPeriod=14d), est vraiment une erreur et doit être corrigée à mon avis.
Quand au log de l’erreur, là tout se discute :)